### ISSUE TITLE
[USERSTORY] Generation-D Signature in Project

### ISSUE THREAD

**Issue by Interperle** [Generation-D/application-platform#188]
1. As an developer
1. I want to sign the Open Source Project for Generation-D
1. So that it can be verifies as Project of Generation-D

### Acceptance Criteria

- [ ] Project is signed
- [ ] Developers are mentioned
- [ ] Generation-D is mentioned

### Dependency

The following Issues must be done first: None

**Comment by Interperle** [comment]
Won't Do
